dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -61,12 +58,12 @@ JWT_NULL_SIGNING_KEY_WITH_ERROR.useraction=Do one of the following actions: 1) I
 
 # Do not translate "JSON Web Token", "JWT", "jwtBuilder"
 JWT_BUILDER_INVALID=CWWKS6008E: The specified JSON Web Token (JWT) builder ID [{0}] is not valid. Verify that a JWT builder with the specified ID is configured.
-JWT_BUILDER_INVALID.explanation=A JSON web token (JWT) builder with the specified ID does not exist in the server configuration.
+JWT_BUILDER_INVALID.explanation=A JSON Web Token (JWT) builder with the specified ID does not exist in the server configuration.
 JWT_BUILDER_INVALID.useraction=In the server.xml file, configure a jwtBuilder element with the specified ID.
 
 # Do not translate "JSON Web Token", "JWT"
-JWT_INVALID_CLAIM_VALUE_ERR=CWWKS6009E: The [{1}] value in the JSON web token (JWT) claim [{0}] is not valid.
-JWT_INVALID_CLAIM_VALUE_ERR.explanation=The specified value in the JSON web token (JWT) claim does not meet the JWT specification.
+JWT_INVALID_CLAIM_VALUE_ERR=CWWKS6009E: The [{1}] value in the JSON Web Token (JWT) claim [{0}] is not valid.
```

Comment:
Change `the JSON Web Token (JWT) claim [{0}]` to `the [{0}] JSON Web Token (JWT) claim` unless the variable has to be after the noun.